compiler/src/dotty/tools/dotc/transform/Splicer.scala

```diff
@@ -45,16 +45,6 @@ object Splicer {
         interpretedExpr.fold(tree)(x => PickledQuotes.quotedExprToTree(x))
       }
       catch {
-        case ex: scala.quoted.QuoteError =>
-          val pos1 = ex.from match {
-            case None => pos
-            case Some(expr) =>
-              val reflect: scala.tasty.Reflection = ReflectionImpl(ctx)
-              import reflect._
-              expr.unseal.underlyingArgument.pos.asInstanceOf[SourcePosition]
-          }
-          ctx.error(ex.getMessage, pos1)
-          EmptyTree
```

Comment:
It seems we cannot remove this until we deprecate `QuoteError`?

Reply:
If someone happens to use this exception by mistake it will still print the error message but additionally will print the stack trace.

Reply:
As all other exceptions do